Cargo.toml

```
@@ -22,12 +22,13 @@ comfy-table = "7.1"
 dirs = "5.0"
 
 # solana
+solana-system-interface = { version = "3.0.0", features = ["bincode"] }
 solana-vote-program = "3"
-solana-instruction = "3"
+solana-instruction = "3.1.0"
```

```suggestion
solana-instruction = "3"
```

lets do this for others so we keep 3 as major version for solana across all solana crates